ecosystem/typescript/sdk/src/plugins/token_client.ts

```diff
@@ -309,6 +310,7 @@ export class TokenClient {
     amount: AnyNumber,
     propertyVersion: AnyNumber = 0,
     extraArgs?: OptionalTransactionArgs,
+    fee_payer?: AptosAccount,
```

Comment:
this is a dev exp nit. 
most people don't care about the `extraArgs` so if anyone wants to use `fee_payer` would need to call this function like that (as you probably noticed when writing the test)
```
directTransferToken(..,..,..,..,..,..,..,..,undefined,bob)
```
this can be annoying for the end user. maybe a better design should be here, but for now I wonder if we could make another function `directTransferTokenWithFeePayer()` that has a required `fee_payer` argument and leave this function as is